### PR TITLE
Doc/introduce steps stats

### PR DESCRIPTION
**Pull request by L-M-Sherlock**
This discussion is useful to refer: https://forums.ankiweb.net/t/suggestion-about-the-learning-steps-after-analyzing-the-anki-dataset/51648/90?u=l.m.sherlock

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,7 +139,11 @@ Interpretation:
 
 ### Steps Stats
 
-TODO
+![image](https://github.com/user-attachments/assets/7d7afafb-27f5-4986-b6eb-b5ef486c50a1)
+
+The Steps Stats are based on all cards in your deck or collection (whichever is selected) that you have ever reviewed twice and the first review should be done in last 1 month/year or deck life.
```
Comment (Expertium) [inline README.md:144]:
```suggestion
The Steps Stats are based on all cards in your deck or collection (whichever is selected) that you have ever reviewed twice, and the first review was in the last 1 month/year or deck life.
```

File: README.md [hunk 2]
```diff
@@ -139,7 +139,11 @@ Interpretation:
...
+
+This stats is helpful if you want to fine-tune your (re)learning steps to make the short-term schedule satisify your desired retention. For the mechanism about steps, please see the *Learning Steps* section in [Deck Options - Anki Manual](https://docs.ankiweb.net/deck-options.html#learning-steps).
```
Comment (Expertium) [inline README.md:146]:
```suggestion
These stats are helpful if you want to fine-tune your (re)learning steps to configure your short-term schedule according to your desired retention. If you want to learn more about learning steps, please read [this section of the Anki manual](https://docs.ankiweb.net/deck-options.html#learning-steps).
```

**Review by Expertium — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -139,7 +139,11 @@ Interpretation:
 
 ### Steps Stats
 
-TODO
+![image](https://github.com/user-attachments/assets/7d7afafb-27f5-4986-b6eb-b5ef486c50a1)
+
+The Steps Stats are based on all cards in your deck or collection (whichever is selected) that you have ever reviewed twice, and the first review was in the last 1 month/year or deck life.
```
Comment (user1823) [inline README.md:144]:
```suggestion
The Steps Stats are based on the cards from the selected deck or collection that were first reviewed in the last 1 month/year or deck life and have at least two reviews.
```

File: README.md [hunk 4]
```diff
@@ -139,7 +139,11 @@ Interpretation:
...
+
```
Comment (user1823) [inline README.md:145]:
```suggestion
These stats are helpful for fine-tuning your (re)learning steps to achieve your desired retention in the short-term reviews. To learn more about learning steps, please read [learning steps](https://docs.ankiweb.net/deck-options.html#learning-steps) in the  Anki manual.
```

File: README.md [hunk 5]
```diff
@@ -139,7 +139,11 @@ Interpretation:
...
+These stats are helpful if you want to fine-tune your (re)learning steps to configure your short-term schedule according to your desired retention. If you want to learn more about learning steps, please read [this section of the Anki manual](https://docs.ankiweb.net/deck-options.html#learning-steps).
```
Comment (user1823) [inline README.md:146]:
```suggestion
These stats are helpful for fine-tuning your (re)learning steps to achieve your desired retention in the short-term reviews. To learn more about learning steps, please read [learning steps](https://docs.ankiweb.net/deck-options.html#learning-steps) in the  Anki manual.
```

**Comment by L-M-Sherlock** [comment]
Thanks for you both!

**Pull request merged**